docs/framework/deployment/index.md

```diff
@@ -39,9 +39,9 @@ This article helps you get started deploying the .NET Framework with your applic
   
     -   Installer options:  
   
-        -   [Web installer](../../../docs/framework/getting-started/install-the-net-framework.md#choices)  
+        -   [Web installer](~/docs/framework/install/guide-for-developers.md#installation-choices)  
```

Comment:
Gotcha ... I don't think the period will make it into the `id` tho, so I'll use `#to-install-or-download-the-net-framework-redistributable`.